scripts/perftune.py

```diff
@@ -738,21 +763,12 @@ def __max_rx_queue_count(self, iface):
         ixgbevf: VF NICs support up to 4 RSS queues.
         i40e:    PF NICs support up to 64 RSS queues.
         i40evf:  VF NICs support up to 16 RSS queues.
+        mlx:     PF/VF NICs throttle to 2 due to si problems when cpu is shared with scylla
```

Comment:
You want to reduce the number of HW queues to the number of IRQ CPUs.
It's not always 2.

Reply:
correct, will be dynamic per cpu tuning

Reply:
Note that this may be any number since we have `--irq-cpu-mask`